### DISCUSSION TITLE
:revolving_hearts: Model topology, collective workflow dynamics observed from 879 seminar

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#82] [people; relating]
"This presentation is also the series of mistakes I made" - from one seminar presentation

Modelers build model (with computer's help) and their morale as below should taken into account for workflow.

1. model expansion dynamics (problem-solving joy addiction): problem discovery - solving - more complex model - problem discovery 

2. paying it forward dynamics (collective debugging): help friend's debugging - face and share bug - asking for friend's help in debugging

**Comment by tomfid** [comment]
I think this is an interesting line of thinking. Here's a related idea: https://metasd.com/2018/06/model-quality-high-road/

**Reply by hyunjimoon** [reply]
From Liza Keating's critique manual, the following part 

> Family Member Test: To what degree is the model generalizable to a family of similar situations, i.e. externally valid? Are the model structures generic? How well is the model calibrated to real-world data?

reminds me of topology of model from [Bayesian workflow](https://arxiv.org/pdf/2011.01808.pdf) paper. (sec.7.4). [This](ryanb.me) provides the following services 

![image](https://user-images.githubusercontent.com/30194633/201230609-b337a939-ee55-43c4-83f3-057ccf0d24da.jpeg)

**Comment by hyunjimoon** [comment]
FAMILY concept and its use for simulation-testing for social science.

FAMILY of models 
-- e.g. MDP, prior sharing models
-- how can estimation be benefited from similar generator structure? Are you aware of any sharing structure for estimation within vensim e.g. can any MCMC result from vensim generator with WIP adjustment =2 aid MCMC for generator with WIP adjustment=3? 
-- distance between models can be measured to determine which model to explore next (Bayesian workflow paper sec.7.4 topology of models). We can use RL here.

**Reply by tomfid** [reply]
> Are you aware of any sharing structure for estimation within vensim e.g. can any MCMC result from vensim generator with WIP adjustment =2 aid MCMC for generator with WIP adjustment=3? 

Seems like a great thing to do, but it's not obvious to me how to do it. I guess one simple option would be to provide a way to seed the chains in a new MCMC run with the sample from a previous one.
